packages/vite/src/node/plugins/reporter.ts

```diff
@@ -149,7 +155,7 @@ export function buildReporterPlugin(config: ResolvedConfig): Plugin {
                     group: isCSS ? 'CSS' : 'Assets',
                     size: chunk.source.length,
                     mapSize: null, // Rollup doesn't support CSS maps?
-                    compressedSize: isCSS
+                    compressedSize: isCompressibleFile(chunk.fileName)
```

Comment:
```suggestion
                    compressedSize: isCSS || isCompressibleAsset(chunk.fileName)
```
(So you can remove js and css form the list)